src/routes/personal/personal-phone-numbers-change-routes.js

```diff
@@ -0,0 +1,54 @@
+import { personalPhoneSchema } from '../../schemas/personal/personal-phone-schema.js'
+import { formatValidationErrors } from '../../utils/format-validation-errors.js'
+import { BAD_REQUEST } from '../../constants/status-codes.js'
+import { personalPhoneNumbersChangePresenter } from '../../presenters/personal/personal-phone-numbers-change-presenter.js'
+import { fetchPersonalChangeService } from '../../services/personal/fetch-personal-change-service.js'
+import { setSessionData } from '../../utils/session/set-session-data.js'
+
+const getPersonalPhoneNumbersChange = {
+  method: 'GET',
+  path: '/account-phone-numbers-change',
+  handler: async (request, h) => {
+    const { yar, auth } = request
+    const personalDetails = await fetchPersonalChangeService(yar, auth.credentials, 'changePersonalPhoneNumbers')
```

Comment:
Something in the change route isn't working. 
If you enter a phone number and then click change on the check page the nuber just entered isnt replaying